### PR TITLE
allow to customize docker-compose.yml using ~/.pndev_config.toml

### PR DESCRIPTION
**Pull request by ghedamat**

### CODE REVIEW

**Comment by ghedamat** [comment]
@typeoneerror if this is merged and released we can write a custom docker config for you that doesn't run redis and I will install redis on your host machine instead, not ideal but I'm desperate because I don't understand what else can be broken in your ubuntu setup

File: README.md [hunk 1]
```diff
@@ -29,6 +29,19 @@ pndev doctor
 
 ```
 install_path = 'DEV/PN'
+docker_compose_path = 'my-local-docker-config' # NOT NEEDED
```
Comment (rcy) [inline README.md:32]: this should be

```
docker_compose_path = 'my-local-docker-config/docker-compose.yml'
```

**Review by rcy — CHANGES_REQUESTED** [review]
documentation about config change requested

**Comment by rcy** [comment]
tested by editing `.pndev_config.toml` copied catalog to ~/my/local-docker-config, commented out mail catcher in docker-compose.yml, ran `cargo run up` and everything came up except mailcatcher, as expected

File: README.md (hunk 1 above)
Comment (ghedamat) [reply README.md:32]: Good catch thank you! Will fix

**Comment by ghedamat** [comment]
bors merge

**Pull request merged**